src/config.ts

```diff
@@ -65,24 +65,24 @@ export const globalOptions = Args.create(
   'This script is an automated turn-burning script for the Kingdom of Loathing that spends a day\'s resources and adventures on farming\n\
 You can use multiple options in conjunction, e.g. "garbo nobarf ascend"',
   {
-    nobarf: Args.flag({
+    ascend: Args.flag({
       setting: "",
-      help: "do beginning of the day setup, embezzlers, and various daily flags, but will terminate before normal Barf Mountain turns. May close NEP for the day.",
+      help: "operate under the assumption that you're ascending after running it, rather than experiencing rollover. It will use borrowed time, it won't charge stinky cheese items, etc.",
       default: false,
     }),
-    ascend: Args.flag({
+    dumpstash: Args.flag({
       setting: "",
-      help: "operate under the assumption that you're ascending after running it, rather than experiencing rollover. It will use borrowed time, it won't charge stinky cheese items, etc.",
+      help: "return all items to the clan stash, then quit",
```

Comment:
`dumpstash` sounds ambiguous, why not `returnstash`?
Which clan stash? I think help should clarify.